### PR TITLE
:new: Asn2 Part2 Testing

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Tell them to run the tests for the provided code. 

### Why
So they know everything is working. Also, just experience running the unit tests.

### CODE REVIEW

File: src/site/asn2.rst [hunk 1]
```diff
@@ -82,6 +82,15 @@ read through this before really jumping into coding.
 Part 2 --- Run Unit Tests
 =========================
 
+* Run all the tests for the provided completed classes
+* All these tests should pass
```
Comment (twentylemon) [inline src/site/asn2.rst:86]: They won't be able to run anything without fixing the compiler errors first. `DfsMazeSolver` has some abstract methods with no implementation.

Comment (jameshughes89) [reply src/site/asn2.rst]: Hmmmmm. The simplest way to address this for them is for me to add 

```java
public Set<Cell> solve(Maze maze) {
    return null;
}
```
Or maybe im being too paranoid on their behalf Whatdoyathink?

Comment (twentylemon) [reply src/site/asn2.rst]: Intellij should be able to do the same for them too. Could go either way.

File: src/site/asn2.rst [hunk 2]
```diff
@@ -82,6 +82,21 @@ read through this before really jumping into coding.
 Part 2 --- Run Unit Tests
 =========================
 
+* Run all the tests for the provided completed classes
+    * Everything except the ``DfsMazeSolverTest``
+
+* In order to run these, you will need to address the error caused by the ``DfsMazeSolver`` not implementing the ``MazeSolver`` interface
+    * You can comment this out for now, or add an empty ``solve`` method returning ``null`` to make it stop complaining for now
+    * In fact, if press Alt+Enter on the error in ``DfsMazeSolver``, intelliJ will add the empty method for you
```
Comment (twentylemon) [inline src/site/asn2.rst:90]: Remove.

Comment (jameshughes89) [reply src/site/asn2.rst]: the trick is for me to push the commit

**Pull request merged**